starter-code/main.js

```diff
@@ -0,0 +1,117 @@
+
+const MongoDB = require('mongodb');
+const mongoClient = MongoDB.MongoClient;
+const clear = require('clear');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const url = `mongodb://localhost:27017/crunchbase`;
+
+mongoClient.connect(url, (error, db) => {
+  if (error) {
+    console.log('Error trying to connect to the Database');
+    console.log(error);
+  } else {
+    console.log('Connection established correctly!! 😬');
+
+    function mainMenu () {
+      clear();
+      printMenu();
+      rl.question('Type an option: ', (option) => {
+        switch (option) {
+        case '1':
+          db.collection('companies').find({}, {name: 1, _id: 0}).toArray((error, result) => {
+            if (error) {
+              console.log(error);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+            } else {
+              console.log(result);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+            }
+          });
+          break;
+        case '2':
+          db.collection('companies').find({}, {name: 1, _id: 0}).count((error, result) => {
+            if (error) {
+              console.log(error);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+            } else {
+              console.log(result);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+            }
+          });
+          break;
+        case '3':
+          db.collection('companies').find({founded_year: 2004}).count((error, result) => {
+            if (error) {
+              console.log(error);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+            } else {
+              console.log(result);
+              rl.question(`\nType enter to continue: `, (answer) => { mainMenu(); });
+            }
+          });
+          break;
+        case '4':
+          db.collection('companies').find({$and: [{founded_month: 2}, {founded_year: 2004}]}).count((error, result) => {
```

Comment:
You don't need the $and here, when you put more than one query (for different criteria) separated by commas, a logic and is applied. In addition, here you want the list of companies, not to count them.